include/picongpu/plugins/radiation/Radiation.hpp

```diff
@@ -119,6 +120,7 @@ class Radiation : public ISimulationPlugin
     bool radPerGPU;
     std::string folderRadPerGPU;
     DataSpace<simDim> lastGPUpos;
+    int numJobs = 2;
```

Comment:
Why do you initialize this variable with `2` if you set it by default to two later one?

Reply:
I think it would be better to set it via the boost option - there a documentation is also intrinsically given. But this is only my opinion.